benchmarks/simulator.py

```diff
@@ -45,3 +46,26 @@ def time_scipy_lsoda_no_compiler_directives(self):
 
     def time_cupsoda(self):
         self.sim_cupsoda.run()
+
+
+class SSASchloegl(object):
+    def setup(self):
+        self.nsims = 100
+        self.timer = timeit.default_timer
+        self.model = schloegl.model
+        self.tspan = np.linspace(0, 100, 101)
+        self.stochkit_sim = StochKitSimulator(self.model, tspan=self.tspan)
+        self.cuda_ssa_sim = CudaSSASimulator(self.model, tspan=self.tspan)
+        self.opencl_ssa_sim = OpenCLSSASimulator(self.model, tspan=self.tspan)
```

Comment:
Updated. Is expensive since it has to compile everything the first time.

Reply:
Check the ASV docs on `setup_cache` -- it needs to return something, and the `time_*` methods also need to be refactored to accept that same thing. I think this benchmark will error as written now, or it might just give weird/wrong results. You can go back to using plain `setup()` and it will be slow but correct, and we can resolve this some other time.